### ISSUE TITLE
java.lang.StackOverflowError: missing stacktrace

### ISSUE THREAD

**Issue by Swaroop-Cybrosys** [wt-io-it/odoo-pycharm-plugin#74]
Plugin Version: 0.6.6
IntelliJ Version: PyCharm 2022.2.3
IntelliJ Build: PY-222.4345.23

<details><summary>java.lang.StackOverflowError</summary>

```
Please copy the stacktrace/file to the issue manually (the text is too big to post in a link)
```
</details>

**Comment by ap-wtioit** [comment]
@Swaroop-Cybrosys any chance you can reproduce this and add the full stack here? otherwise i cannot investigate this and reliably fix it.

**Comment by ap-wtioit** [comment]
One possible StackOverflowError discovered while fixing #99:
* having recursive dependencies in modules (module1 depends on module2 which depends on module1)

**Comment by bigbear3001** [comment]
this might have been fixed in 0.6.9 (pending JetBrains approval). if you are eager to test the new version, you can use the jar file from https://github.com/wt-io-it/odoo-pycharm-plugin/tree/v0.6.9

but as the stacktrace is missing i cannot be sure it is fixed.
